python311Packages.coffea: 2024.2.2 -> 2024.4.1

GaetanLepage opened the pull request:
## Description of changes

- `dask-expr`: init at 1.0.12 (https://github.com/dask/dask-expr)
- `distributed`: 2023.12.0 -> 2024.4.2 (https://github.com/dask/distributed/blob/2024.4.2/docs/source/changelog.rst)
- `dask`: 2024.1.1 -> 2024.4.2 (https://docs.dask.org/en/latest/changelog.html)
- `coffea`: 2024.2.2 -> 2024.4.1 (https://github.com/CoffeaTeam/coffea/releases/tag/v2024.4.0)

cc @FRidh @veprbl 

## Things done

<!-- Please check what applies. Note that these are not hard requirements but merely serve as information for reviewers. -->

- Built on platform(s)
  - [x] x86_64-linux
  - [x] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- For non-Linux: Is sandboxing enabled in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
  - [ ] `sandbox = relaxed`
  - [ ] `sandbox = true`
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [x] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [ ] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [24.05 Release Notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2405.section.md) (or backporting [23.05](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2305.section.md) and [23.11](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2311.section.md) Release notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

<!--
To help with the large amounts of pull requests, we would appreciate your
reviews of other pull requests, especially simple package updates. Just leave a
comment describing what you have tested in the relevant package/service.
Reviewing helps to reduce the average time-to-merge for everyone.
Thanks a lot if you do!

List of open PRs: https://github.com/NixOS/nixpkgs/pulls
Reviewing guidelines: https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#reviewing-contributions
-->

---

Add a :+1: [reaction] to [pull requests you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[pull requests you find important]: https://github.com/NixOS/nixpkgs/pulls?q=is%3Aopen+sort%3Areactions-%2B1-desc

GaetanLepage committed:
python311Packages.coffea: 2024.2.2 -> 2024.4.1

Diff:
https://github.com/CoffeaTeam/coffea/compare/refs/tags/v2024.2.2...v2024.4.1

Changelog: https://github.com/CoffeaTeam/coffea/releases/tag/v2024.4.1

GaetanLepage committed:
python311Packages.dask: 2024.1.1 -> 2024.4.2

Diff: https://github.com/dask/dask/compare/refs/tags/2024.1.1...2024.4.2

Changelog: https://docs.dask.org/en/latest/changelog.html

GaetanLepage committed:
python311Packages.dask-expr: init at 1.0.12

kirillrdy commented on pkgs/development/python-modules/dask-expr/default.nix:
```diff
@@ -0,0 +1,57 @@
+
+  postPatch = ''
+    substituteInPlace pyproject.toml \
+      --replace "versioneer[toml]==0.28" "versioneer[toml]"
```
```suggestion
      --replace-fail "versioneer[toml]==0.28" "versioneer[toml]"
```

GaetanLepage replied:
Done.

kirillrdy commented on pkgs/development/python-modules/dask-expr/default.nix:
```diff
@@ -0,0 +1,57 @@
+    description = "";
+    homepage = "https://github.com/dask/dask-expr";
+    license = licenses.bsd3;
+    maintainers = with maintainers; [ ];
```
meta.maintainers must be set for new packages.

GaetanLepage replied:
Done. I also formatted the file with the new RFC formatter.

veprbl commented:
> ERROR dask - ValueError: Must install dask-expr to activate query planning.

seen on ofborg

GaetanLepage commented:
> ERROR dask - ValueError: Must install dask-expr to activate query planning.

Indeed, the [dask](https://github.com/dask/dask) package now has a new dependency: [dask-expr](https://github.com/dask/dask-expr).
They both depend on each other.
I dont kbow what approach is the best to deal with this circular dependency.

It seems to be a `dask-expr` seems to hardly depend on `dask` while `dask` _only_ needs `dask-expr` for some of its features.
I planned to disable the tests in `dask` that are failing because of this...
Would you adopt a `passthru.tests = ...` approach ?

veprbl commented:
Yes, let's disable related tests in dask.

GaetanLepage commented:
> Yes, let's disable related tests in dask.

Not that easy in the end. It gets triggered during the test collection phase: https://github.com/dask/dask/blob/ba24c0aec4fc2e86bf6d6191234a47e340c835bb/dask/conftest.py#L6

veprbl commented:
Is that not catching ImportError?

veprbl commented:
If it's rethrown as ValueError, just fix that?

GaetanLepage commented:
> If it's rethrown as ValueError, just fix that?

Yes:
```
ERROR dask - ValueError: Must install dask-expr to activate query planning.
```
I'll try to fix it.

GaetanLepage commented:
[Patching the import works fine](https://github.com/NixOS/nixpkgs/pull/303043/files#diff-9230e700646bd87e325a1afe5db7ff7341e26678a99c5b3d2dfc136fa8685990R124-R126).
However, [the number of tests that are now failing is enormous](https://gist.github.com/GaetanLepage/1c2900d59efaa21726920fdcb71aa848).
It doesn't look sensible to me to manually list them in `disabledTestPaths`...

veprbl commented:
We might need to ask the upstream for advice.

GaetanLepage committed:
python311Packages.distributed: 2023.12.0 -> 2024.4.2

Diff: https://github.com/dask/distributed/compare/refs/tags/2023.12.0...2024.4.2

Changelog: https://github.com/dask/distributed/blob/2024.4.2/docs/source/changelog.rst

GaetanLepage commented:
I finally managed to make everything build.
I went with a `passthru.tests` approach for handling the circular dependency issue in `dask`.

GaetanLepage commented:
Result of `nixpkgs-review pr 303043` run on aarch64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>10 packages marked as broken and skipped:</summary>
  <ul>
    <li>python311Packages.clifford</li>
    <li>python311Packages.clifford.dist</li>
    <li>python311Packages.stytra</li>
    <li>python311Packages.stytra.dist</li>
    <li>python311Packages.sunpy</li>
    <li>python311Packages.sunpy.dist</li>
    <li>python312Packages.clifford</li>
    <li>python312Packages.clifford.dist</li>
    <li>python312Packages.sunpy</li>
    <li>python312Packages.sunpy.dist</li>
  </ul>
</details>
<details>
  <summary>64 packages failed to build:</summary>
  <ul>
    <li>deface</li>
    <li>deface.dist</li>
    <li>napari (python311Packages.napari)</li>
    <li>napari.dist (python311Packages.napari.dist)</li>
    <li>ome_zarr (python311Packages.ome-zarr)</li>
    <li>ome_zarr.dist (python311Packages.ome-zarr.dist)</li>
    <li>python311Packages.dask-image</li>
    <li>python311Packages.dask-image.dist</li>
    <li>python311Packages.devito</li>
    <li>python311Packages.devito.dist</li>
    <li>python311Packages.fastai</li>
    <li>python311Packages.fastai.dist</li>
    <li>python311Packages.insightface</li>
    <li>python311Packages.insightface.dist</li>
    <li>python311Packages.mmcv</li>
    <li>python311Packages.mmcv.dist</li>
    <li>python311Packages.niworkflows</li>
    <li>python311Packages.niworkflows.dist</li>
    <li>python311Packages.spacy</li>
    <li>python311Packages.spacy-lookups-data</li>
    <li>python311Packages.spacy-lookups-data.dist</li>
    <li>python311Packages.spacy-transformers</li>
    <li>python311Packages.spacy-transformers.dist</li>
    <li>python311Packages.spacy.dist</li>
    <li>python311Packages.streamz</li>
    <li>python311Packages.streamz.dist</li>
    <li>python311Packages.textacy</li>
    <li>python311Packages.textacy.dist</li>
    <li>python311Packages.textnets</li>
    <li>python311Packages.textnets.dist</li>
    <li>python311Packages.torchrl</li>
    <li>python311Packages.torchrl.dist</li>
    <li>python312Packages.dask-image</li>
    <li>python312Packages.dask-image.dist</li>
    <li>python312Packages.dask-jobqueue</li>
    <li>python312Packages.dask-jobqueue.dist</li>
    <li>python312Packages.dask-yarn</li>
    <li>python312Packages.dask-yarn.dist</li>
    <li>python312Packages.devito</li>
    <li>python312Packages.devito.dist</li>
    <li>python312Packages.glymur</li>
    <li>python312Packages.glymur.dist</li>
    <li>python312Packages.gymnasium</li>
    <li>python312Packages.gymnasium.dist</li>
    <li>python312Packages.insightface</li>
    <li>python312Packages.insightface.dist</li>
    <li>python312Packages.intake-parquet</li>
    <li>python312Packages.intake-parquet.dist</li>
    <li>python312Packages.mmcv</li>
    <li>python312Packages.mmcv.dist</li>
    <li>python312Packages.ome-zarr</li>
    <li>python312Packages.ome-zarr.dist</li>
    <li>python312Packages.pyfftw</li>
    <li>python312Packages.pyfftw.dist</li>
    <li>python312Packages.sfepy</li>
    <li>python312Packages.sfepy.dist</li>
    <li>python312Packages.spectral-cube</li>
    <li>python312Packages.spectral-cube.dist</li>
    <li>python312Packages.streamz</li>
    <li>python312Packages.streamz.dist</li>
    <li>python312Packages.torchrl</li>
    <li>python312Packages.torchrl.dist</li>
    <li>python312Packages.tsfresh</li>
    <li>python312Packages.tsfresh.dist</li>
  </ul>
</details>
<details>
  <summary>230 packages built:</summary>
  <ul>
    <li>backgroundremover</li>
    <li>backgroundremover.dist</li>
    <li>csvkit</li>
    <li>csvkit.dist</li>
    <li>easyocr (python311Packages.easyocr)</li>
    <li>easyocr.dist (python311Packages.easyocr.dist)</li>
    <li>home-assistant-component-tests.dremel_3d_printer</li>
    <li>home-assistant-component-tests.environment_canada</li>
    <li>hydrus</li>
    <li>hydrus.doc</li>
    <li>python311Packages.agate-sql</li>
    <li>python311Packages.agate-sql.dist</li>
    <li>python311Packages.albumentations</li>
    <li>python311Packages.albumentations.dist</li>
    <li>python311Packages.aplpy</li>
    <li>python311Packages.aplpy.dist</li>
    <li>python311Packages.batchgenerators</li>
    <li>python311Packages.batchgenerators.dist</li>
    <li>python311Packages.bsuite</li>
    <li>python311Packages.bsuite.dist</li>
    <li>python311Packages.casa-formats-io</li>
    <li>python311Packages.casa-formats-io.dist</li>
    <li>python311Packages.cf-xarray</li>
    <li>python311Packages.cf-xarray.dist</li>
    <li>python311Packages.coffea</li>
    <li>python311Packages.coffea.dist</li>
    <li>python311Packages.crate</li>
    <li>python311Packages.crate.dist</li>
    <li>python311Packages.dask</li>
    <li>python311Packages.dask-awkward</li>
    <li>python311Packages.dask-awkward.dist</li>
    <li>python311Packages.dask-expr</li>
    <li>python311Packages.dask-expr.dist</li>
    <li>python311Packages.dask-gateway</li>
    <li>python311Packages.dask-gateway.dist</li>
    <li>python311Packages.dask-glm</li>
    <li>python311Packages.dask-glm.dist</li>
    <li>python311Packages.dask-histogram</li>
    <li>python311Packages.dask-histogram.dist</li>
    <li>python311Packages.dask-jobqueue</li>
    <li>python311Packages.dask-jobqueue.dist</li>
    <li>python311Packages.dask-ml</li>
    <li>python311Packages.dask-ml.dist</li>
    <li>python311Packages.dask-mpi</li>
    <li>python311Packages.dask-mpi.dist</li>
    <li>python311Packages.dask-yarn</li>
    <li>python311Packages.dask-yarn.dist</li>
    <li>python311Packages.dask.dist</li>
    <li>python311Packages.datashader</li>
    <li>python311Packages.datashader.dist</li>
    <li>python311Packages.distributed</li>
    <li>python311Packages.distributed.dist</li>
    <li>python311Packages.dremel3dpy</li>
    <li>python311Packages.dremel3dpy.dist</li>
    <li>python311Packages.env-canada</li>
    <li>python311Packages.env-canada.dist</li>
    <li>python311Packages.glueviz</li>
    <li>python311Packages.glueviz.dist</li>
    <li>python311Packages.glymur</li>
    <li>python311Packages.glymur.dist</li>
    <li>python311Packages.gradio</li>
    <li>python311Packages.gradio.dist</li>
    <li>python311Packages.gymnasium</li>
    <li>python311Packages.gymnasium.dist</li>
    <li>python311Packages.handout</li>
    <li>python311Packages.handout.dist</li>
    <li>python311Packages.imagecorruptions</li>
    <li>python311Packages.imagecorruptions.dist</li>
    <li>python311Packages.imageio</li>
    <li>python311Packages.imageio.dist</li>
    <li>python311Packages.intake</li>
    <li>python311Packages.intake-parquet</li>
    <li>python311Packages.intake-parquet.dist</li>
    <li>python311Packages.intake.dist</li>
    <li>python311Packages.k-diffusion</li>
    <li>python311Packages.k-diffusion.dist</li>
    <li>python311Packages.lime</li>
    <li>python311Packages.lime.dist</li>
    <li>python311Packages.moviepy</li>
    <li>python311Packages.moviepy.dist</li>
    <li>python311Packages.mung</li>
    <li>python311Packages.mung.dist</li>
    <li>python311Packages.muscima</li>
    <li>python311Packages.muscima.dist</li>
    <li>python311Packages.napari-console</li>
    <li>python311Packages.napari-console.dist</li>
    <li>python311Packages.napari-svg</li>
    <li>python311Packages.napari-svg.dist</li>
    <li>python311Packages.omrdatasettools</li>
    <li>python311Packages.omrdatasettools.dist</li>
    <li>python311Packages.open-clip-torch</li>
    <li>python311Packages.open-clip-torch.dist</li>
    <li>python311Packages.pims</li>
    <li>python311Packages.pims.dist</li>
    <li>python311Packages.psd-tools</li>
    <li>python311Packages.psd-tools.dist</li>
    <li>python311Packages.pvextractor</li>
    <li>python311Packages.pvextractor.dist</li>
    <li>python311Packages.pyfftw</li>
    <li>python311Packages.pyfftw.dist</li>
    <li>python311Packages.pyrender</li>
    <li>python311Packages.pyrender.dist</li>
    <li>python311Packages.pyvista</li>
    <li>python311Packages.pyvista.dist</li>
    <li>python311Packages.qcodes</li>
    <li>python311Packages.qcodes-contrib-drivers</li>
    <li>python311Packages.qcodes-contrib-drivers.dist</li>
    <li>python311Packages.qcodes.dist</li>
    <li>python311Packages.reproject</li>
    <li>python311Packages.reproject.dist</li>
    <li>python311Packages.scikit-image</li>
    <li>python311Packages.scikit-image.dist</li>
    <li>python311Packages.sfepy</li>
    <li>python311Packages.sfepy.dist</li>
    <li>python311Packages.skrl</li>
    <li>python311Packages.skrl.dist</li>
    <li>python311Packages.slicedimage</li>
    <li>python311Packages.slicedimage.dist</li>
    <li>python311Packages.spacy-loggers</li>
    <li>python311Packages.spacy-loggers.dist</li>
    <li>python311Packages.sparse</li>
    <li>python311Packages.sparse.dist</li>
    <li>python311Packages.spectral-cube</li>
    <li>python311Packages.spectral-cube.dist</li>
    <li>python311Packages.stumpy</li>
    <li>python311Packages.stumpy.dist</li>
    <li>python311Packages.test-tube</li>
    <li>python311Packages.test-tube.dist</li>
    <li>python311Packages.tifffile</li>
    <li>python311Packages.tifffile.dist</li>
    <li>python311Packages.tsfresh</li>
    <li>python311Packages.tsfresh.dist</li>
    <li>python311Packages.wandb</li>
    <li>python311Packages.wandb.dist</li>
    <li>python311Packages.webdataset</li>
    <li>python311Packages.webdataset.dist</li>
    <li>python312Packages.agate-sql</li>
    <li>python312Packages.agate-sql.dist</li>
    <li>python312Packages.albumentations</li>
    <li>python312Packages.albumentations.dist</li>
    <li>python312Packages.aplpy</li>
    <li>python312Packages.aplpy.dist</li>
    <li>python312Packages.batchgenerators</li>
    <li>python312Packages.batchgenerators.dist</li>
    <li>python312Packages.bsuite</li>
    <li>python312Packages.bsuite.dist</li>
    <li>python312Packages.casa-formats-io</li>
    <li>python312Packages.casa-formats-io.dist</li>
    <li>python312Packages.cf-xarray</li>
    <li>python312Packages.cf-xarray.dist</li>
    <li>python312Packages.crate</li>
    <li>python312Packages.crate.dist</li>
    <li>python312Packages.dask</li>
    <li>python312Packages.dask-expr</li>
    <li>python312Packages.dask-expr.dist</li>
    <li>python312Packages.dask-gateway</li>
    <li>python312Packages.dask-gateway.dist</li>
    <li>python312Packages.dask-glm</li>
    <li>python312Packages.dask-glm.dist</li>
    <li>python312Packages.dask-histogram</li>
    <li>python312Packages.dask-histogram.dist</li>
    <li>python312Packages.dask-ml</li>
    <li>python312Packages.dask-ml.dist</li>
    <li>python312Packages.dask-mpi</li>
    <li>python312Packages.dask-mpi.dist</li>
    <li>python312Packages.dask.dist</li>
    <li>python312Packages.distributed</li>
    <li>python312Packages.distributed.dist</li>
    <li>python312Packages.dremel3dpy</li>
    <li>python312Packages.dremel3dpy.dist</li>
    <li>python312Packages.easyocr</li>
    <li>python312Packages.easyocr.dist</li>
    <li>python312Packages.env-canada</li>
    <li>python312Packages.env-canada.dist</li>
    <li>python312Packages.gradio</li>
    <li>python312Packages.gradio.dist</li>
    <li>python312Packages.handout</li>
    <li>python312Packages.handout.dist</li>
    <li>python312Packages.imagecorruptions</li>
    <li>python312Packages.imagecorruptions.dist</li>
    <li>python312Packages.imageio</li>
    <li>python312Packages.imageio.dist</li>
    <li>python312Packages.lime</li>
    <li>python312Packages.lime.dist</li>
    <li>python312Packages.moviepy</li>
    <li>python312Packages.moviepy.dist</li>
    <li>python312Packages.mung</li>
    <li>python312Packages.mung.dist</li>
    <li>python312Packages.muscima</li>
    <li>python312Packages.muscima.dist</li>
    <li>python312Packages.napari-svg</li>
    <li>python312Packages.napari-svg.dist</li>
    <li>python312Packages.omrdatasettools</li>
    <li>python312Packages.omrdatasettools.dist</li>
    <li>python312Packages.open-clip-torch</li>
    <li>python312Packages.open-clip-torch.dist</li>
    <li>python312Packages.pims</li>
    <li>python312Packages.pims.dist</li>
    <li>python312Packages.psd-tools</li>
    <li>python312Packages.psd-tools.dist</li>
    <li>python312Packages.pyrender</li>
    <li>python312Packages.pyrender.dist</li>
    <li>python312Packages.pyvista</li>
    <li>python312Packages.pyvista.dist</li>
    <li>python312Packages.qcodes</li>
    <li>python312Packages.qcodes-contrib-drivers</li>
    <li>python312Packages.qcodes-contrib-drivers.dist</li>
    <li>python312Packages.qcodes.dist</li>
    <li>python312Packages.reproject</li>
    <li>python312Packages.reproject.dist</li>
    <li>python312Packages.scikit-image</li>
    <li>python312Packages.scikit-image.dist</li>
    <li>python312Packages.slicedimage</li>
    <li>python312Packages.slicedimage.dist</li>
    <li>python312Packages.sparse</li>
    <li>python312Packages.sparse.dist</li>
    <li>python312Packages.stumpy</li>
    <li>python312Packages.stumpy.dist</li>
    <li>python312Packages.test-tube</li>
    <li>python312Packages.test-tube.dist</li>
    <li>python312Packages.tifffile</li>
    <li>python312Packages.tifffile.dist</li>
    <li>python312Packages.webdataset</li>
    <li>python312Packages.webdataset.dist</li>
    <li>rclip</li>
    <li>rclip.dist</li>
    <li>tartube</li>
    <li>tartube-yt-dlp</li>
    <li>tartube-yt-dlp.dist</li>
    <li>tartube.dist</li>
  </ul>
</details>

GaetanLepage commented:
Result of `nixpkgs-review pr 303043` run on aarch64-darwin [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>36 packages marked as broken and skipped:</summary>
  <ul>
    <li>napari</li>
    <li>napari.dist</li>
    <li>ome_zarr</li>
    <li>ome_zarr.dist</li>
    <li>python311Packages.bsuite</li>
    <li>python311Packages.bsuite.dist</li>
    <li>python311Packages.clifford</li>
    <li>python311Packages.clifford.dist</li>
    <li>python311Packages.dask-jobqueue</li>
    <li>python311Packages.dask-jobqueue.dist</li>
    <li>python311Packages.dask-yarn</li>
    <li>python311Packages.dask-yarn.dist</li>
    <li>python311Packages.napari</li>
    <li>python311Packages.napari.dist</li>
    <li>python311Packages.ome-zarr</li>
    <li>python311Packages.ome-zarr.dist</li>
    <li>python311Packages.streamz</li>
    <li>python311Packages.streamz.dist</li>
    <li>python311Packages.stytra</li>
    <li>python311Packages.stytra.dist</li>
    <li>python311Packages.torchrl</li>
    <li>python311Packages.torchrl.dist</li>
    <li>python312Packages.bsuite</li>
    <li>python312Packages.bsuite.dist</li>
    <li>python312Packages.clifford</li>
    <li>python312Packages.clifford.dist</li>
    <li>python312Packages.dask-jobqueue</li>
    <li>python312Packages.dask-jobqueue.dist</li>
    <li>python312Packages.dask-yarn</li>
    <li>python312Packages.dask-yarn.dist</li>
    <li>python312Packages.ome-zarr</li>
    <li>python312Packages.ome-zarr.dist</li>
    <li>python312Packages.streamz</li>
    <li>python312Packages.streamz.dist</li>
    <li>python312Packages.torchrl</li>
    <li>python312Packages.torchrl.dist</li>
  </ul>
</details>
<details>
  <summary>92 packages failed to build:</summary>
  <ul>
    <li>csvkit</li>
    <li>csvkit.dist</li>
    <li>python311Packages.coffea</li>
    <li>python311Packages.coffea.dist</li>
    <li>python311Packages.dask-awkward</li>
    <li>python311Packages.dask-awkward.dist</li>
    <li>python311Packages.dask-expr</li>
    <li>python311Packages.dask-expr.dist</li>
    <li>python311Packages.dask-glm</li>
    <li>python311Packages.dask-glm.dist</li>
    <li>python311Packages.dask-image</li>
    <li>python311Packages.dask-image.dist</li>
    <li>python311Packages.dask-ml</li>
    <li>python311Packages.dask-ml.dist</li>
    <li>python311Packages.devito</li>
    <li>python311Packages.devito.dist</li>
    <li>python311Packages.fastai</li>
    <li>python311Packages.fastai.dist</li>
    <li>python311Packages.gradio</li>
    <li>python311Packages.gradio.dist</li>
    <li>python311Packages.k-diffusion</li>
    <li>python311Packages.k-diffusion.dist</li>
    <li>python311Packages.mmcv</li>
    <li>python311Packages.mmcv.dist</li>
    <li>python311Packages.niworkflows</li>
    <li>python311Packages.niworkflows.dist</li>
    <li>python311Packages.open-clip-torch</li>
    <li>python311Packages.open-clip-torch.dist</li>
    <li>python311Packages.pyfftw</li>
    <li>python311Packages.pyfftw.dist</li>
    <li>python311Packages.qcodes</li>
    <li>python311Packages.qcodes-contrib-drivers</li>
    <li>python311Packages.qcodes-contrib-drivers.dist</li>
    <li>python311Packages.qcodes.dist</li>
    <li>python311Packages.sfepy</li>
    <li>python311Packages.sfepy.dist</li>
    <li>python311Packages.skrl</li>
    <li>python311Packages.skrl.dist</li>
    <li>python311Packages.spacy</li>
    <li>python311Packages.spacy-loggers</li>
    <li>python311Packages.spacy-loggers.dist</li>
    <li>python311Packages.spacy-lookups-data</li>
    <li>python311Packages.spacy-lookups-data.dist</li>
    <li>python311Packages.spacy-transformers</li>
    <li>python311Packages.spacy-transformers.dist</li>
    <li>python311Packages.spacy.dist</li>
    <li>python311Packages.textacy</li>
    <li>python311Packages.textacy.dist</li>
    <li>python311Packages.textnets</li>
    <li>python311Packages.textnets.dist</li>
    <li>python311Packages.tsfresh</li>
    <li>python311Packages.tsfresh.dist</li>
    <li>python311Packages.wandb</li>
    <li>python311Packages.wandb.dist</li>
    <li>python311Packages.webdataset</li>
    <li>python311Packages.webdataset.dist</li>
    <li>python312Packages.dask-expr</li>
    <li>python312Packages.dask-expr.dist</li>
    <li>python312Packages.dask-glm</li>
    <li>python312Packages.dask-glm.dist</li>
    <li>python312Packages.dask-image</li>
    <li>python312Packages.dask-image.dist</li>
    <li>python312Packages.dask-ml</li>
    <li>python312Packages.dask-ml.dist</li>
    <li>python312Packages.devito</li>
    <li>python312Packages.devito.dist</li>
    <li>python312Packages.glymur</li>
    <li>python312Packages.glymur.dist</li>
    <li>python312Packages.gradio</li>
    <li>python312Packages.gradio.dist</li>
    <li>python312Packages.intake-parquet</li>
    <li>python312Packages.intake-parquet.dist</li>
    <li>python312Packages.mmcv</li>
    <li>python312Packages.mmcv.dist</li>
    <li>python312Packages.open-clip-torch</li>
    <li>python312Packages.open-clip-torch.dist</li>
    <li>python312Packages.pyfftw</li>
    <li>python312Packages.pyfftw.dist</li>
    <li>python312Packages.qcodes</li>
    <li>python312Packages.qcodes-contrib-drivers</li>
    <li>python312Packages.qcodes-contrib-drivers.dist</li>
    <li>python312Packages.qcodes.dist</li>
    <li>python312Packages.sfepy</li>
    <li>python312Packages.sfepy.dist</li>
    <li>python312Packages.spectral-cube</li>
    <li>python312Packages.spectral-cube.dist</li>
    <li>python312Packages.tsfresh</li>
    <li>python312Packages.tsfresh.dist</li>
    <li>python312Packages.webdataset</li>
    <li>python312Packages.webdataset.dist</li>
    <li>rclip</li>
    <li>rclip.dist</li>
  </ul>
</details>
<details>
  <summary>162 packages built:</summary>
  <ul>
    <li>backgroundremover</li>
    <li>backgroundremover.dist</li>
    <li>deface</li>
    <li>deface.dist</li>
    <li>easyocr (python311Packages.easyocr)</li>
    <li>easyocr.dist (python311Packages.easyocr.dist)</li>
    <li>hydrus</li>
    <li>hydrus.doc</li>
    <li>python311Packages.agate-sql</li>
    <li>python311Packages.agate-sql.dist</li>
    <li>python311Packages.albumentations</li>
    <li>python311Packages.albumentations.dist</li>
    <li>python311Packages.aplpy</li>
    <li>python311Packages.aplpy.dist</li>
    <li>python311Packages.batchgenerators</li>
    <li>python311Packages.batchgenerators.dist</li>
    <li>python311Packages.casa-formats-io</li>
    <li>python311Packages.casa-formats-io.dist</li>
    <li>python311Packages.cf-xarray</li>
    <li>python311Packages.cf-xarray.dist</li>
    <li>python311Packages.crate</li>
    <li>python311Packages.crate.dist</li>
    <li>python311Packages.dask</li>
    <li>python311Packages.dask-gateway</li>
    <li>python311Packages.dask-gateway.dist</li>
    <li>python311Packages.dask-histogram</li>
    <li>python311Packages.dask-histogram.dist</li>
    <li>python311Packages.dask-mpi</li>
    <li>python311Packages.dask-mpi.dist</li>
    <li>python311Packages.dask.dist</li>
    <li>python311Packages.datashader</li>
    <li>python311Packages.datashader.dist</li>
    <li>python311Packages.distributed</li>
    <li>python311Packages.distributed.dist</li>
    <li>python311Packages.dremel3dpy</li>
    <li>python311Packages.dremel3dpy.dist</li>
    <li>python311Packages.env-canada</li>
    <li>python311Packages.env-canada.dist</li>
    <li>python311Packages.glueviz</li>
    <li>python311Packages.glueviz.dist</li>
    <li>python311Packages.glymur</li>
    <li>python311Packages.glymur.dist</li>
    <li>python311Packages.handout</li>
    <li>python311Packages.handout.dist</li>
    <li>python311Packages.imagecorruptions</li>
    <li>python311Packages.imagecorruptions.dist</li>
    <li>python311Packages.imageio</li>
    <li>python311Packages.imageio.dist</li>
    <li>python311Packages.insightface</li>
    <li>python311Packages.insightface.dist</li>
    <li>python311Packages.intake</li>
    <li>python311Packages.intake-parquet</li>
    <li>python311Packages.intake-parquet.dist</li>
    <li>python311Packages.intake.dist</li>
    <li>python311Packages.lime</li>
    <li>python311Packages.lime.dist</li>
    <li>python311Packages.moviepy</li>
    <li>python311Packages.moviepy.dist</li>
    <li>python311Packages.mung</li>
    <li>python311Packages.mung.dist</li>
    <li>python311Packages.muscima</li>
    <li>python311Packages.muscima.dist</li>
    <li>python311Packages.napari-console</li>
    <li>python311Packages.napari-console.dist</li>
    <li>python311Packages.napari-svg</li>
    <li>python311Packages.napari-svg.dist</li>
    <li>python311Packages.omrdatasettools</li>
    <li>python311Packages.omrdatasettools.dist</li>
    <li>python311Packages.pims</li>
    <li>python311Packages.pims.dist</li>
    <li>python311Packages.psd-tools</li>
    <li>python311Packages.psd-tools.dist</li>
    <li>python311Packages.pvextractor</li>
    <li>python311Packages.pvextractor.dist</li>
    <li>python311Packages.pyvista</li>
    <li>python311Packages.pyvista.dist</li>
    <li>python311Packages.reproject</li>
    <li>python311Packages.reproject.dist</li>
    <li>python311Packages.scikit-image</li>
    <li>python311Packages.scikit-image.dist</li>
    <li>python311Packages.slicedimage</li>
    <li>python311Packages.slicedimage.dist</li>
    <li>python311Packages.sparse</li>
    <li>python311Packages.sparse.dist</li>
    <li>python311Packages.spectral-cube</li>
    <li>python311Packages.spectral-cube.dist</li>
    <li>python311Packages.stumpy</li>
    <li>python311Packages.stumpy.dist</li>
    <li>python311Packages.test-tube</li>
    <li>python311Packages.test-tube.dist</li>
    <li>python311Packages.tifffile</li>
    <li>python311Packages.tifffile.dist</li>
    <li>python312Packages.agate-sql</li>
    <li>python312Packages.agate-sql.dist</li>
    <li>python312Packages.albumentations</li>
    <li>python312Packages.albumentations.dist</li>
    <li>python312Packages.aplpy</li>
    <li>python312Packages.aplpy.dist</li>
    <li>python312Packages.batchgenerators</li>
    <li>python312Packages.batchgenerators.dist</li>
    <li>python312Packages.casa-formats-io</li>
    <li>python312Packages.casa-formats-io.dist</li>
    <li>python312Packages.cf-xarray</li>
    <li>python312Packages.cf-xarray.dist</li>
    <li>python312Packages.crate</li>
    <li>python312Packages.crate.dist</li>
    <li>python312Packages.dask</li>
    <li>python312Packages.dask-gateway</li>
    <li>python312Packages.dask-gateway.dist</li>
    <li>python312Packages.dask-histogram</li>
    <li>python312Packages.dask-histogram.dist</li>
    <li>python312Packages.dask-mpi</li>
    <li>python312Packages.dask-mpi.dist</li>
    <li>python312Packages.dask.dist</li>
    <li>python312Packages.distributed</li>
    <li>python312Packages.distributed.dist</li>
    <li>python312Packages.dremel3dpy</li>
    <li>python312Packages.dremel3dpy.dist</li>
    <li>python312Packages.easyocr</li>
    <li>python312Packages.easyocr.dist</li>
    <li>python312Packages.env-canada</li>
    <li>python312Packages.env-canada.dist</li>
    <li>python312Packages.handout</li>
    <li>python312Packages.handout.dist</li>
    <li>python312Packages.imagecorruptions</li>
    <li>python312Packages.imagecorruptions.dist</li>
    <li>python312Packages.imageio</li>
    <li>python312Packages.imageio.dist</li>
    <li>python312Packages.insightface</li>
    <li>python312Packages.insightface.dist</li>
    <li>python312Packages.lime</li>
    <li>python312Packages.lime.dist</li>
    <li>python312Packages.moviepy</li>
    <li>python312Packages.moviepy.dist</li>
    <li>python312Packages.mung</li>
    <li>python312Packages.mung.dist</li>
    <li>python312Packages.muscima</li>
    <li>python312Packages.muscima.dist</li>
    <li>python312Packages.napari-svg</li>
    <li>python312Packages.napari-svg.dist</li>
    <li>python312Packages.omrdatasettools</li>
    <li>python312Packages.omrdatasettools.dist</li>
    <li>python312Packages.pims</li>
    <li>python312Packages.pims.dist</li>
    <li>python312Packages.psd-tools</li>
    <li>python312Packages.psd-tools.dist</li>
    <li>python312Packages.pyvista</li>
    <li>python312Packages.pyvista.dist</li>
    <li>python312Packages.reproject</li>
    <li>python312Packages.reproject.dist</li>
    <li>python312Packages.scikit-image</li>
    <li>python312Packages.scikit-image.dist</li>
    <li>python312Packages.slicedimage</li>
    <li>python312Packages.slicedimage.dist</li>
    <li>python312Packages.sparse</li>
    <li>python312Packages.sparse.dist</li>
    <li>python312Packages.stumpy</li>
    <li>python312Packages.stumpy.dist</li>
    <li>python312Packages.test-tube</li>
    <li>python312Packages.test-tube.dist</li>
    <li>python312Packages.tifffile</li>
    <li>python312Packages.tifffile.dist</li>
  </ul>
</details>

GaetanLepage commented:
Result of `nixpkgs-review pr 303043` run on x86_64-darwin [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>36 packages marked as broken and skipped:</summary>
  <ul>
    <li>napari</li>
    <li>napari.dist</li>
    <li>ome_zarr</li>
    <li>ome_zarr.dist</li>
    <li>python311Packages.bsuite</li>
    <li>python311Packages.bsuite.dist</li>
    <li>python311Packages.clifford</li>
    <li>python311Packages.clifford.dist</li>
    <li>python311Packages.dask-jobqueue</li>
    <li>python311Packages.dask-jobqueue.dist</li>
    <li>python311Packages.dask-yarn</li>
    <li>python311Packages.dask-yarn.dist</li>
    <li>python311Packages.napari</li>
    <li>python311Packages.napari.dist</li>
    <li>python311Packages.ome-zarr</li>
    <li>python311Packages.ome-zarr.dist</li>
    <li>python311Packages.streamz</li>
    <li>python311Packages.streamz.dist</li>
    <li>python311Packages.stytra</li>
    <li>python311Packages.stytra.dist</li>
    <li>python311Packages.torchrl</li>
    <li>python311Packages.torchrl.dist</li>
    <li>python312Packages.bsuite</li>
    <li>python312Packages.bsuite.dist</li>
    <li>python312Packages.clifford</li>
    <li>python312Packages.clifford.dist</li>
    <li>python312Packages.dask-jobqueue</li>
    <li>python312Packages.dask-jobqueue.dist</li>
    <li>python312Packages.dask-yarn</li>
    <li>python312Packages.dask-yarn.dist</li>
    <li>python312Packages.ome-zarr</li>
    <li>python312Packages.ome-zarr.dist</li>
    <li>python312Packages.streamz</li>
    <li>python312Packages.streamz.dist</li>
    <li>python312Packages.torchrl</li>
    <li>python312Packages.torchrl.dist</li>
  </ul>
</details>
<details>
  <summary>84 packages failed to build:</summary>
  <ul>
    <li>csvkit</li>
    <li>csvkit.dist</li>
    <li>python311Packages.coffea</li>
    <li>python311Packages.coffea.dist</li>
    <li>python311Packages.dask-awkward</li>
    <li>python311Packages.dask-awkward.dist</li>
    <li>python311Packages.dask-expr</li>
    <li>python311Packages.dask-expr.dist</li>
    <li>python311Packages.dask-glm</li>
    <li>python311Packages.dask-glm.dist</li>
    <li>python311Packages.dask-image</li>
    <li>python311Packages.dask-image.dist</li>
    <li>python311Packages.dask-ml</li>
    <li>python311Packages.dask-ml.dist</li>
    <li>python311Packages.devito</li>
    <li>python311Packages.devito.dist</li>
    <li>python311Packages.fastai</li>
    <li>python311Packages.fastai.dist</li>
    <li>python311Packages.gradio</li>
    <li>python311Packages.gradio.dist</li>
    <li>python311Packages.mmcv</li>
    <li>python311Packages.mmcv.dist</li>
    <li>python311Packages.niworkflows</li>
    <li>python311Packages.niworkflows.dist</li>
    <li>python311Packages.open-clip-torch</li>
    <li>python311Packages.open-clip-torch.dist</li>
    <li>python311Packages.pyfftw</li>
    <li>python311Packages.pyfftw.dist</li>
    <li>python311Packages.qcodes</li>
    <li>python311Packages.qcodes-contrib-drivers</li>
    <li>python311Packages.qcodes-contrib-drivers.dist</li>
    <li>python311Packages.qcodes.dist</li>
    <li>python311Packages.sfepy</li>
    <li>python311Packages.sfepy.dist</li>
    <li>python311Packages.spacy</li>
    <li>python311Packages.spacy-lookups-data</li>
    <li>python311Packages.spacy-lookups-data.dist</li>
    <li>python311Packages.spacy-transformers</li>
    <li>python311Packages.spacy-transformers.dist</li>
    <li>python311Packages.spacy.dist</li>
    <li>python311Packages.textacy</li>
    <li>python311Packages.textacy.dist</li>
    <li>python311Packages.textnets</li>
    <li>python311Packages.textnets.dist</li>
    <li>python311Packages.tsfresh</li>
    <li>python311Packages.tsfresh.dist</li>
    <li>python311Packages.webdataset</li>
    <li>python311Packages.webdataset.dist</li>
    <li>python312Packages.dask-expr</li>
    <li>python312Packages.dask-expr.dist</li>
    <li>python312Packages.dask-glm</li>
    <li>python312Packages.dask-glm.dist</li>
    <li>python312Packages.dask-image</li>
    <li>python312Packages.dask-image.dist</li>
    <li>python312Packages.dask-ml</li>
    <li>python312Packages.dask-ml.dist</li>
    <li>python312Packages.devito</li>
    <li>python312Packages.devito.dist</li>
    <li>python312Packages.glymur</li>
    <li>python312Packages.glymur.dist</li>
    <li>python312Packages.gradio</li>
    <li>python312Packages.gradio.dist</li>
    <li>python312Packages.intake-parquet</li>
    <li>python312Packages.intake-parquet.dist</li>
    <li>python312Packages.mmcv</li>
    <li>python312Packages.mmcv.dist</li>
    <li>python312Packages.open-clip-torch</li>
    <li>python312Packages.open-clip-torch.dist</li>
    <li>python312Packages.pyfftw</li>
    <li>python312Packages.pyfftw.dist</li>
    <li>python312Packages.qcodes</li>
    <li>python312Packages.qcodes-contrib-drivers</li>
    <li>python312Packages.qcodes-contrib-drivers.dist</li>
    <li>python312Packages.qcodes.dist</li>
    <li>python312Packages.sfepy</li>
    <li>python312Packages.sfepy.dist</li>
    <li>python312Packages.spectral-cube</li>
    <li>python312Packages.spectral-cube.dist</li>
    <li>python312Packages.tsfresh</li>
    <li>python312Packages.tsfresh.dist</li>
    <li>python312Packages.webdataset</li>
    <li>python312Packages.webdataset.dist</li>
    <li>rclip</li>
    <li>rclip.dist</li>
  </ul>
</details>
<details>
  <summary>170 packages built:</summary>
  <ul>
    <li>backgroundremover</li>
    <li>backgroundremover.dist</li>
    <li>deface</li>
    <li>deface.dist</li>
    <li>easyocr (python311Packages.easyocr)</li>
    <li>easyocr.dist (python311Packages.easyocr.dist)</li>
    <li>hydrus</li>
    <li>hydrus.doc</li>
    <li>python311Packages.agate-sql</li>
    <li>python311Packages.agate-sql.dist</li>
    <li>python311Packages.albumentations</li>
    <li>python311Packages.albumentations.dist</li>
    <li>python311Packages.aplpy</li>
    <li>python311Packages.aplpy.dist</li>
    <li>python311Packages.batchgenerators</li>
    <li>python311Packages.batchgenerators.dist</li>
    <li>python311Packages.casa-formats-io</li>
    <li>python311Packages.casa-formats-io.dist</li>
    <li>python311Packages.cf-xarray</li>
    <li>python311Packages.cf-xarray.dist</li>
    <li>python311Packages.crate</li>
    <li>python311Packages.crate.dist</li>
    <li>python311Packages.dask</li>
    <li>python311Packages.dask-gateway</li>
    <li>python311Packages.dask-gateway.dist</li>
    <li>python311Packages.dask-histogram</li>
    <li>python311Packages.dask-histogram.dist</li>
    <li>python311Packages.dask-mpi</li>
    <li>python311Packages.dask-mpi.dist</li>
    <li>python311Packages.dask.dist</li>
    <li>python311Packages.datashader</li>
    <li>python311Packages.datashader.dist</li>
    <li>python311Packages.distributed</li>
    <li>python311Packages.distributed.dist</li>
    <li>python311Packages.dremel3dpy</li>
    <li>python311Packages.dremel3dpy.dist</li>
    <li>python311Packages.env-canada</li>
    <li>python311Packages.env-canada.dist</li>
    <li>python311Packages.glueviz</li>
    <li>python311Packages.glueviz.dist</li>
    <li>python311Packages.glymur</li>
    <li>python311Packages.glymur.dist</li>
    <li>python311Packages.handout</li>
    <li>python311Packages.handout.dist</li>
    <li>python311Packages.imagecorruptions</li>
    <li>python311Packages.imagecorruptions.dist</li>
    <li>python311Packages.imageio</li>
    <li>python311Packages.imageio.dist</li>
    <li>python311Packages.insightface</li>
    <li>python311Packages.insightface.dist</li>
    <li>python311Packages.intake</li>
    <li>python311Packages.intake-parquet</li>
    <li>python311Packages.intake-parquet.dist</li>
    <li>python311Packages.intake.dist</li>
    <li>python311Packages.k-diffusion</li>
    <li>python311Packages.k-diffusion.dist</li>
    <li>python311Packages.lime</li>
    <li>python311Packages.lime.dist</li>
    <li>python311Packages.moviepy</li>
    <li>python311Packages.moviepy.dist</li>
    <li>python311Packages.mung</li>
    <li>python311Packages.mung.dist</li>
    <li>python311Packages.muscima</li>
    <li>python311Packages.muscima.dist</li>
    <li>python311Packages.napari-console</li>
    <li>python311Packages.napari-console.dist</li>
    <li>python311Packages.napari-svg</li>
    <li>python311Packages.napari-svg.dist</li>
    <li>python311Packages.omrdatasettools</li>
    <li>python311Packages.omrdatasettools.dist</li>
    <li>python311Packages.pims</li>
    <li>python311Packages.pims.dist</li>
    <li>python311Packages.psd-tools</li>
    <li>python311Packages.psd-tools.dist</li>
    <li>python311Packages.pvextractor</li>
    <li>python311Packages.pvextractor.dist</li>
    <li>python311Packages.pyvista</li>
    <li>python311Packages.pyvista.dist</li>
    <li>python311Packages.reproject</li>
    <li>python311Packages.reproject.dist</li>
    <li>python311Packages.scikit-image</li>
    <li>python311Packages.scikit-image.dist</li>
    <li>python311Packages.skrl</li>
    <li>python311Packages.skrl.dist</li>
    <li>python311Packages.slicedimage</li>
    <li>python311Packages.slicedimage.dist</li>
    <li>python311Packages.spacy-loggers</li>
    <li>python311Packages.spacy-loggers.dist</li>
    <li>python311Packages.sparse</li>
    <li>python311Packages.sparse.dist</li>
    <li>python311Packages.spectral-cube</li>
    <li>python311Packages.spectral-cube.dist</li>
    <li>python311Packages.stumpy</li>
    <li>python311Packages.stumpy.dist</li>
    <li>python311Packages.test-tube</li>
    <li>python311Packages.test-tube.dist</li>
    <li>python311Packages.tifffile</li>
    <li>python311Packages.tifffile.dist</li>
    <li>python311Packages.wandb</li>
    <li>python311Packages.wandb.dist</li>
    <li>python312Packages.agate-sql</li>
    <li>python312Packages.agate-sql.dist</li>
    <li>python312Packages.albumentations</li>
    <li>python312Packages.albumentations.dist</li>
    <li>python312Packages.aplpy</li>
    <li>python312Packages.aplpy.dist</li>
    <li>python312Packages.batchgenerators</li>
    <li>python312Packages.batchgenerators.dist</li>
    <li>python312Packages.casa-formats-io</li>
    <li>python312Packages.casa-formats-io.dist</li>
    <li>python312Packages.cf-xarray</li>
    <li>python312Packages.cf-xarray.dist</li>
    <li>python312Packages.crate</li>
    <li>python312Packages.crate.dist</li>
    <li>python312Packages.dask</li>
    <li>python312Packages.dask-gateway</li>
    <li>python312Packages.dask-gateway.dist</li>
    <li>python312Packages.dask-histogram</li>
    <li>python312Packages.dask-histogram.dist</li>
    <li>python312Packages.dask-mpi</li>
    <li>python312Packages.dask-mpi.dist</li>
    <li>python312Packages.dask.dist</li>
    <li>python312Packages.distributed</li>
    <li>python312Packages.distributed.dist</li>
    <li>python312Packages.dremel3dpy</li>
    <li>python312Packages.dremel3dpy.dist</li>
    <li>python312Packages.easyocr</li>
    <li>python312Packages.easyocr.dist</li>
    <li>python312Packages.env-canada</li>
    <li>python312Packages.env-canada.dist</li>
    <li>python312Packages.handout</li>
    <li>python312Packages.handout.dist</li>
    <li>python312Packages.imagecorruptions</li>
    <li>python312Packages.imagecorruptions.dist</li>
    <li>python312Packages.imageio</li>
    <li>python312Packages.imageio.dist</li>
    <li>python312Packages.insightface</li>
    <li>python312Packages.insightface.dist</li>
    <li>python312Packages.lime</li>
    <li>python312Packages.lime.dist</li>
    <li>python312Packages.moviepy</li>
    <li>python312Packages.moviepy.dist</li>
    <li>python312Packages.mung</li>
    <li>python312Packages.mung.dist</li>
    <li>python312Packages.muscima</li>
    <li>python312Packages.muscima.dist</li>
    <li>python312Packages.napari-svg</li>
    <li>python312Packages.napari-svg.dist</li>
    <li>python312Packages.omrdatasettools</li>
    <li>python312Packages.omrdatasettools.dist</li>
    <li>python312Packages.pims</li>
    <li>python312Packages.pims.dist</li>
    <li>python312Packages.psd-tools</li>
    <li>python312Packages.psd-tools.dist</li>
    <li>python312Packages.pyvista</li>
    <li>python312Packages.pyvista.dist</li>
    <li>python312Packages.reproject</li>
    <li>python312Packages.reproject.dist</li>
    <li>python312Packages.scikit-image</li>
    <li>python312Packages.scikit-image.dist</li>
    <li>python312Packages.slicedimage</li>
    <li>python312Packages.slicedimage.dist</li>
    <li>python312Packages.sparse</li>
    <li>python312Packages.sparse.dist</li>
    <li>python312Packages.stumpy</li>
    <li>python312Packages.stumpy.dist</li>
    <li>python312Packages.test-tube</li>
    <li>python312Packages.test-tube.dist</li>
    <li>python312Packages.tifffile</li>
    <li>python312Packages.tifffile.dist</li>
  </ul>
</details>

GaetanLepage commented:
Result of `nixpkgs-review pr 303043` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>18 packages marked as broken and skipped:</summary>
  <ul>
    <li>python311Packages.caffeWithCuda</li>
    <li>python311Packages.caffeWithCuda.bin</li>
    <li>python311Packages.clifford</li>
    <li>python311Packages.clifford.dist</li>
    <li>python311Packages.dm-sonnet</li>
    <li>python311Packages.dm-sonnet.dist</li>
    <li>python311Packages.stytra</li>
    <li>python311Packages.stytra.dist</li>
    <li>python311Packages.sunpy</li>
    <li>python311Packages.sunpy.dist</li>
    <li>python311Packages.tensorflow-datasets</li>
    <li>python311Packages.tensorflow-datasets.dist</li>
    <li>python312Packages.clifford</li>
    <li>python312Packages.clifford.dist</li>
    <li>python312Packages.sunpy</li>
    <li>python312Packages.sunpy.dist</li>
    <li>tts</li>
    <li>tts.dist</li>
  </ul>
</details>
<details>
  <summary>66 packages failed to build:</summary>
  <ul>
    <li>napari (python311Packages.napari)</li>
    <li>napari.dist (python311Packages.napari.dist)</li>
    <li>ome_zarr (python311Packages.ome-zarr)</li>
    <li>ome_zarr.dist (python311Packages.ome-zarr.dist)</li>
    <li>python311Packages.dask-image</li>
    <li>python311Packages.dask-image.dist</li>
    <li>python311Packages.fastai</li>
    <li>python311Packages.fastai.dist</li>
    <li>python311Packages.mmcv</li>
    <li>python311Packages.mmcv.dist</li>
    <li>python311Packages.niworkflows</li>
    <li>python311Packages.niworkflows.dist</li>
    <li>python311Packages.qcodes</li>
    <li>python311Packages.qcodes-contrib-drivers</li>
    <li>python311Packages.qcodes-contrib-drivers.dist</li>
    <li>python311Packages.qcodes.dist</li>
    <li>python311Packages.spacy</li>
    <li>python311Packages.spacy-lookups-data</li>
    <li>python311Packages.spacy-lookups-data.dist</li>
    <li>python311Packages.spacy-transformers</li>
    <li>python311Packages.spacy-transformers.dist</li>
    <li>python311Packages.spacy.dist</li>
    <li>python311Packages.streamz</li>
    <li>python311Packages.streamz.dist</li>
    <li>python311Packages.textacy</li>
    <li>python311Packages.textacy.dist</li>
    <li>python311Packages.textnets</li>
    <li>python311Packages.textnets.dist</li>
    <li>python311Packages.torchrl</li>
    <li>python311Packages.torchrl.dist</li>
    <li>python312Packages.dask-image</li>
    <li>python312Packages.dask-image.dist</li>
    <li>python312Packages.dask-jobqueue</li>
    <li>python312Packages.dask-jobqueue.dist</li>
    <li>python312Packages.dask-yarn</li>
    <li>python312Packages.dask-yarn.dist</li>
    <li>python312Packages.devito</li>
    <li>python312Packages.devito.dist</li>
    <li>python312Packages.glymur</li>
    <li>python312Packages.glymur.dist</li>
    <li>python312Packages.gradio</li>
    <li>python312Packages.gradio.dist</li>
    <li>python312Packages.gymnasium</li>
    <li>python312Packages.gymnasium.dist</li>
    <li>python312Packages.intake-parquet</li>
    <li>python312Packages.intake-parquet.dist</li>
    <li>python312Packages.mmcv</li>
    <li>python312Packages.mmcv.dist</li>
    <li>python312Packages.ome-zarr</li>
    <li>python312Packages.ome-zarr.dist</li>
    <li>python312Packages.pyfftw</li>
    <li>python312Packages.pyfftw.dist</li>
    <li>python312Packages.qcodes</li>
    <li>python312Packages.qcodes-contrib-drivers</li>
    <li>python312Packages.qcodes-contrib-drivers.dist</li>
    <li>python312Packages.qcodes.dist</li>
    <li>python312Packages.sfepy</li>
    <li>python312Packages.sfepy.dist</li>
    <li>python312Packages.spectral-cube</li>
    <li>python312Packages.spectral-cube.dist</li>
    <li>python312Packages.streamz</li>
    <li>python312Packages.streamz.dist</li>
    <li>python312Packages.torchrl</li>
    <li>python312Packages.torchrl.dist</li>
    <li>python312Packages.tsfresh</li>
    <li>python312Packages.tsfresh.dist</li>
  </ul>
</details>
<details>
  <summary>234 packages built:</summary>
  <ul>
    <li>backgroundremover</li>
    <li>backgroundremover.dist</li>
    <li>csvkit</li>
    <li>csvkit.dist</li>
    <li>deface</li>
    <li>deface.dist</li>
    <li>easyocr (python311Packages.easyocr)</li>
    <li>easyocr.dist (python311Packages.easyocr.dist)</li>
    <li>home-assistant-component-tests.dremel_3d_printer</li>
    <li>home-assistant-component-tests.environment_canada</li>
    <li>hydrus</li>
    <li>hydrus.doc</li>
    <li>intensity-normalization (python311Packages.intensity-normalization)</li>
    <li>intensity-normalization.dist (python311Packages.intensity-normalization.dist)</li>
    <li>python311Packages.agate-sql</li>
    <li>python311Packages.agate-sql.dist</li>
    <li>python311Packages.albumentations</li>
    <li>python311Packages.albumentations.dist</li>
    <li>python311Packages.aplpy</li>
    <li>python311Packages.aplpy.dist</li>
    <li>python311Packages.batchgenerators</li>
    <li>python311Packages.batchgenerators.dist</li>
    <li>python311Packages.bsuite</li>
    <li>python311Packages.bsuite.dist</li>
    <li>python311Packages.caffe</li>
    <li>python311Packages.caffe.bin</li>
    <li>python311Packages.casa-formats-io</li>
    <li>python311Packages.casa-formats-io.dist</li>
    <li>python311Packages.cf-xarray</li>
    <li>python311Packages.cf-xarray.dist</li>
    <li>python311Packages.coffea</li>
    <li>python311Packages.coffea.dist</li>
    <li>python311Packages.crate</li>
    <li>python311Packages.crate.dist</li>
    <li>python311Packages.dalle-mini</li>
    <li>python311Packages.dalle-mini.dist</li>
    <li>python311Packages.dask</li>
    <li>python311Packages.dask-awkward</li>
    <li>python311Packages.dask-awkward.dist</li>
    <li>python311Packages.dask-expr</li>
    <li>python311Packages.dask-expr.dist</li>
    <li>python311Packages.dask-gateway</li>
    <li>python311Packages.dask-gateway.dist</li>
    <li>python311Packages.dask-glm</li>
    <li>python311Packages.dask-glm.dist</li>
    <li>python311Packages.dask-histogram</li>
    <li>python311Packages.dask-histogram.dist</li>
    <li>python311Packages.dask-jobqueue</li>
    <li>python311Packages.dask-jobqueue.dist</li>
    <li>python311Packages.dask-ml</li>
    <li>python311Packages.dask-ml.dist</li>
    <li>python311Packages.dask-mpi</li>
    <li>python311Packages.dask-mpi.dist</li>
    <li>python311Packages.dask-yarn</li>
    <li>python311Packages.dask-yarn.dist</li>
    <li>python311Packages.dask.dist</li>
    <li>python311Packages.datashader</li>
    <li>python311Packages.datashader.dist</li>
    <li>python311Packages.devito</li>
    <li>python311Packages.devito.dist</li>
    <li>python311Packages.distributed</li>
    <li>python311Packages.distributed.dist</li>
    <li>python311Packages.dremel3dpy</li>
    <li>python311Packages.dremel3dpy.dist</li>
    <li>python311Packages.env-canada</li>
    <li>python311Packages.env-canada.dist</li>
    <li>python311Packages.glueviz</li>
    <li>python311Packages.glueviz.dist</li>
    <li>python311Packages.glymur</li>
    <li>python311Packages.glymur.dist</li>
    <li>python311Packages.gradio</li>
    <li>python311Packages.gradio.dist</li>
    <li>python311Packages.gymnasium</li>
    <li>python311Packages.gymnasium.dist</li>
    <li>python311Packages.handout</li>
    <li>python311Packages.handout.dist</li>
    <li>python311Packages.imagecorruptions</li>
    <li>python311Packages.imagecorruptions.dist</li>
    <li>python311Packages.imageio</li>
    <li>python311Packages.imageio.dist</li>
    <li>python311Packages.insightface</li>
    <li>python311Packages.insightface.dist</li>
    <li>python311Packages.intake</li>
    <li>python311Packages.intake-parquet</li>
    <li>python311Packages.intake-parquet.dist</li>
    <li>python311Packages.intake.dist</li>
    <li>python311Packages.k-diffusion</li>
    <li>python311Packages.k-diffusion.dist</li>
    <li>python311Packages.lime</li>
    <li>python311Packages.lime.dist</li>
    <li>python311Packages.moviepy</li>
    <li>python311Packages.moviepy.dist</li>
    <li>python311Packages.mung</li>
    <li>python311Packages.mung.dist</li>
    <li>python311Packages.muscima</li>
    <li>python311Packages.muscima.dist</li>
    <li>python311Packages.napari-console</li>
    <li>python311Packages.napari-console.dist</li>
    <li>python311Packages.napari-svg</li>
    <li>python311Packages.napari-svg.dist</li>
    <li>python311Packages.omrdatasettools</li>
    <li>python311Packages.omrdatasettools.dist</li>
    <li>python311Packages.open-clip-torch</li>
    <li>python311Packages.open-clip-torch.dist</li>
    <li>python311Packages.pims</li>
    <li>python311Packages.pims.dist</li>
    <li>python311Packages.psd-tools</li>
    <li>python311Packages.psd-tools.dist</li>
    <li>python311Packages.pvextractor</li>
    <li>python311Packages.pvextractor.dist</li>
    <li>python311Packages.pyfftw</li>
    <li>python311Packages.pyfftw.dist</li>
    <li>python311Packages.pyrender</li>
    <li>python311Packages.pyrender.dist</li>
    <li>python311Packages.pyvista</li>
    <li>python311Packages.pyvista.dist</li>
    <li>python311Packages.reproject</li>
    <li>python311Packages.reproject.dist</li>
    <li>python311Packages.scikit-image</li>
    <li>python311Packages.scikit-image.dist</li>
    <li>python311Packages.sfepy</li>
    <li>python311Packages.sfepy.dist</li>
    <li>python311Packages.skrl</li>
    <li>python311Packages.skrl.dist</li>
    <li>python311Packages.slicedimage</li>
    <li>python311Packages.slicedimage.dist</li>
    <li>python311Packages.spacy-loggers</li>
    <li>python311Packages.spacy-loggers.dist</li>
    <li>python311Packages.sparse</li>
    <li>python311Packages.sparse.dist</li>
    <li>python311Packages.spectral-cube</li>
    <li>python311Packages.spectral-cube.dist</li>
    <li>python311Packages.stumpy</li>
    <li>python311Packages.stumpy.dist</li>
    <li>python311Packages.test-tube</li>
    <li>python311Packages.test-tube.dist</li>
    <li>python311Packages.tifffile</li>
    <li>python311Packages.tifffile.dist</li>
    <li>python311Packages.tsfresh</li>
    <li>python311Packages.tsfresh.dist</li>
    <li>python311Packages.wandb</li>
    <li>python311Packages.wandb.dist</li>
    <li>python311Packages.webdataset</li>
    <li>python311Packages.webdataset.dist</li>
    <li>python312Packages.agate-sql</li>
    <li>python312Packages.agate-sql.dist</li>
    <li>python312Packages.albumentations</li>
    <li>python312Packages.albumentations.dist</li>
    <li>python312Packages.aplpy</li>
    <li>python312Packages.aplpy.dist</li>
    <li>python312Packages.batchgenerators</li>
    <li>python312Packages.batchgenerators.dist</li>
    <li>python312Packages.bsuite</li>
    <li>python312Packages.bsuite.dist</li>
    <li>python312Packages.casa-formats-io</li>
    <li>python312Packages.casa-formats-io.dist</li>
    <li>python312Packages.cf-xarray</li>
    <li>python312Packages.cf-xarray.dist</li>
    <li>python312Packages.crate</li>
    <li>python312Packages.crate.dist</li>
    <li>python312Packages.dask</li>
    <li>python312Packages.dask-expr</li>
    <li>python312Packages.dask-expr.dist</li>
    <li>python312Packages.dask-gateway</li>
    <li>python312Packages.dask-gateway.dist</li>
    <li>python312Packages.dask-glm</li>
    <li>python312Packages.dask-glm.dist</li>
    <li>python312Packages.dask-histogram</li>
    <li>python312Packages.dask-histogram.dist</li>
    <li>python312Packages.dask-ml</li>
    <li>python312Packages.dask-ml.dist</li>
    <li>python312Packages.dask-mpi</li>
    <li>python312Packages.dask-mpi.dist</li>
    <li>python312Packages.dask.dist</li>
    <li>python312Packages.distributed</li>
    <li>python312Packages.distributed.dist</li>
    <li>python312Packages.dremel3dpy</li>
    <li>python312Packages.dremel3dpy.dist</li>
    <li>python312Packages.easyocr</li>
    <li>python312Packages.easyocr.dist</li>
    <li>python312Packages.env-canada</li>
    <li>python312Packages.env-canada.dist</li>
    <li>python312Packages.handout</li>
    <li>python312Packages.handout.dist</li>
    <li>python312Packages.imagecorruptions</li>
    <li>python312Packages.imagecorruptions.dist</li>
    <li>python312Packages.imageio</li>
    <li>python312Packages.imageio.dist</li>
    <li>python312Packages.insightface</li>
    <li>python312Packages.insightface.dist</li>
    <li>python312Packages.lime</li>
    <li>python312Packages.lime.dist</li>
    <li>python312Packages.moviepy</li>
    <li>python312Packages.moviepy.dist</li>
    <li>python312Packages.mung</li>
    <li>python312Packages.mung.dist</li>
    <li>python312Packages.muscima</li>
    <li>python312Packages.muscima.dist</li>
    <li>python312Packages.napari-svg</li>
    <li>python312Packages.napari-svg.dist</li>
    <li>python312Packages.omrdatasettools</li>
    <li>python312Packages.omrdatasettools.dist</li>
    <li>python312Packages.open-clip-torch</li>
    <li>python312Packages.open-clip-torch.dist</li>
    <li>python312Packages.pims</li>
    <li>python312Packages.pims.dist</li>
    <li>python312Packages.psd-tools</li>
    <li>python312Packages.psd-tools.dist</li>
    <li>python312Packages.pyrender</li>
    <li>python312Packages.pyrender.dist</li>
    <li>python312Packages.pyvista</li>
    <li>python312Packages.pyvista.dist</li>
    <li>python312Packages.reproject</li>
    <li>python312Packages.reproject.dist</li>
    <li>python312Packages.scikit-image</li>
    <li>python312Packages.scikit-image.dist</li>
    <li>python312Packages.slicedimage</li>
    <li>python312Packages.slicedimage.dist</li>
    <li>python312Packages.sparse</li>
    <li>python312Packages.sparse.dist</li>
    <li>python312Packages.stumpy</li>
    <li>python312Packages.stumpy.dist</li>
    <li>python312Packages.test-tube</li>
    <li>python312Packages.test-tube.dist</li>
    <li>python312Packages.tifffile</li>
    <li>python312Packages.tifffile.dist</li>
    <li>python312Packages.webdataset</li>
    <li>python312Packages.webdataset.dist</li>
    <li>rclip</li>
    <li>rclip.dist</li>
    <li>tartube</li>
    <li>tartube-yt-dlp</li>
    <li>tartube-yt-dlp.dist</li>
    <li>tartube.dist</li>
  </ul>
</details>

GaetanLepage committed:
python311Packages.awkward: disable failing test for darwin

veprbl reviewed (approved):
LGTM
